Gemfile

```diff
@@ -173,14 +173,13 @@ group :development, :test do
   # Development server
   gem 'puma', '~> 3.12'
 
-  if ENV['DEBUGGER'] == 'byebug'
-    # Call 'debugger' anywhere in the code to stop execution and get a debugger console
-    gem 'byebug'
-  else
-    # Debug in VS Code
-    gem 'ruby-debug-ide'
-    gem 'debase'
-  end
+  # See config/initializers/04-debugger.rb
+  #
+  # Call 'debugger' anywhere in the code to stop execution and get a debugger console
+  gem 'byebug', require: false
+  # Debug in VS Code
+  gem 'ruby-debug-ide', require: false
+  gem 'debase', require: false
```

Comment:
Switched to installing the both but requiring (activating) them in the running code depending on the environment variable.